install_scripts/fedora_camel_toolbox.sh

```diff
@@ -15,3 +15,8 @@ cd $HOME_DIR
 /opt/karaf/bin/client < "$SHARED_DIR/install_scripts/fedora_camel_toolbox.script"
 
 sed -i 's|solr.baseUrl=localhost:8983/solr/collection1|solr.baseUrl=localhost:8080/solr/collection1|' /opt/karaf/etc/org.fcrepo.camel.indexing.solr.cfg
+if [ "$FEDORA_AUTH" == "true" ]; then
+  sed -i 's|fcrepo.authUsername=$|fcrepo.authUsername=fedoraAdmin|' /opt/karaf/etc/org.fcrepo.camel.indexing.solr.cfg
```

Comment:
This all works as expected, but the admin username and password `sed` commands are also needed for the following:
- org.fcrepo.camel.reindexing.cfg
- org.fcrepo.camel.indexing.triplestore.cfg
- org.fcrepo.camel.fixity.cfg